src/main/kotlin/org/jetbrains/research/ml/coding/assistant/dataset/DatasetFetcher.kt

```diff
@@ -0,0 +1,14 @@
+package org.jetbrains.research.ml.coding.assistant.dataset
+
+import org.jetbrains.research.ml.coding.assistant.dataset.model.Dataset
+import org.jetbrains.research.ml.coding.assistant.dataset.model.TaskSolutions
+import java.io.File
+
+/**
+ * Load dataset model from file
```

Comment:
Please, add a detailed comment about it. I think we should use very detailed comments for all base interfaces. What is the dataset model? Which format is ok for the file?